include/cvodes/cvodes.h

```diff
@@ -297,15 +298,14 @@ SUNDIALS_EXPORT int CVodeGetIntegratorStats(
   void* cvode_mem, long int* nsteps, long int* nfevals, long int* nlinsetups,
   long int* netfails, int* qlast, int* qcur, sunrealtype* hinused,
   sunrealtype* hlast, sunrealtype* hcur, sunrealtype* tcur);
-SUNDIALS_EXPORT int CVodeGetNonlinearSystemData(void* cvode_mem,
-                                                sunrealtype* tcur,
-                                                N_Vector* ypred, N_Vector* yn,
-                                                N_Vector* fn, sunrealtype* gamma,
-                                                sunrealtype* rl1, N_Vector* zn1,
-                                                void** user_data);
+SUNDIALS_EXPORT int CVodeGetNonlinearSystemData(
+  void* cvode_mem, sunrealtype* tcur, N_Vector* ypred_1d, N_Vector* yn_1d,
+  N_Vector* fn_1d, sunrealtype* gamma, sunrealtype* rl1, N_Vector* zn1_1d,
+  void** user_data);
```

Comment:
The vectors are outputs not arrays
```suggestion
  void* cvode_mem, sunrealtype* tcur, N_Vector* ypred, N_Vector* yn,
  N_Vector* fn, sunrealtype* gamma, sunrealtype* rl1, N_Vector* zn1,
  void** user_data);
```